docs/source/advanced_apis/index.md

```diff
@@ -0,0 +1,37 @@
+# Advanced API Providers
```

Comment:
Curious - why "Advanced API Providers" and not just "APIs" ?

Reply:
Yeah I agree, I think Advanced APIs is better